File: src/main/java/com/iflat/pam/bean/PartyBranch.java
package com.iflat.pam.bean;

/**
 * Created by tyriv on 2016/7/29.
 */
public class PartyBranch {

    private String id;
    private String name;
    private String dept;
    private String description;
    private String sequence;

    private String text;
    private String parentId;
    private boolean leaf;

    public PartyBranch() {
        leaf = true;
    }

    public String getText() {
        return text;
    }

    public String getParentId() {
        return parentId;
    }

    public void setParentId(String parentId) {
        this.parentId = parentId;
    }

    public boolean isLeaf() {
        return leaf;
    }

    public void setLeaf(boolean leaf) {
        this.leaf = leaf;
    }

    public String getSequence() {
        return sequence;
    }

    public void setSequence(String sequence) {
        this.sequence = sequence;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
        this.text = name;
    }

    public String getDept() {
        return dept;
    }

    public void setDept(String dept) {
        this.dept = dept;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}

File: web/app/model/pam/PartyBranch.js
Ext.define('iFlat.model.pam.PartyBranch', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'partyBranch.id', mapping: 'id', type: 'string'},
        {name: 'partyBranch.name', mapping: 'name', type: 'string'},
        {name: 'partyBranch.dept', mapping: 'dept', type: 'string'},
        {name: 'partyBranch.description', mapping: 'description', type: 'string'},
        {name: 'partyBranch.sequence', mapping: 'sequence', type: 'string'},
    ]
});